doc/whats_new/v0.23.rst

```diff
@@ -255,6 +255,11 @@ Changelog
   :class:`tree.DecisionTreeRegressor`. :pr:`15864` by
   `Nicolas Hug`_.
 
+- |Feature| :func:`inspection.partial_dependence` and
```

Comment:
If this is not ready for 0.23, we will need to move it to 0.24

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/00426b6af66c7443a603f738ca88701614d66cd8 and https://github.com/scikit-learn/scikit-learn/pull/16619/commits/8c7314ccdea2dbea160a18dc98d772865fd47208